.buildkite/pipeline.yml

```diff
@@ -206,7 +206,7 @@ steps:
 
   - label: "cargo-audit"
     commands:
-      - cargo audit -q
+      - echo "Skipping cargo audit for now, see https://github.com/RustSec/advisory-db/issues/414#issuecomment-702479926" # cargo audit -q
```

Comment:
Can you use the `skip` field instead? You can check an example here: https://github.com/rust-vmm/rust-vmm-ci/blob/c8cf2b70dc21030172f77d5ac3c579c72eacc76e/.buildkite/pipeline.yml#L182

Reply:
Nice. I changed it now.